Update readme to include basic instructions and default docker-compose config gotcha (#25)
SoftwareAG/apama-streaming-analytics-docker-samples: applications/Simple/README.md

either express or implied. 
See the License for the specific language governing permissions and limitations under the License.

Simple
======
This sample is a very simple application, deploying a trivial EPL program into
a running correlator. A container runs the correlator and then runs the client
tools to deploys the program.

If you are running via Kubernetes this sample creates the following
resources which can be accessed via logs and must be deleted via delete:

	* pod simple-sample

Running with Docker-Compose
==============

> Get the Apama Correlator container [here](https://hub.docker.com/_/apama-correlator). More information about building your own image can be found in the parent directory.

Reviewer: Get the Apama Correlator container [here](https://hub.docker.com/_/apama-correlator). More information about building your own image is contained in the README in the parent directory of this sample.
```suggestion
Get the Apama Correlator container [here](https://hub.docker.com/_/apama-correlator). More information about building your own image is contained in the README in the parent directory of this sample.
```

To run with Docker Compose, execute docker-compose from this directory

```
docker-compose up
```

After bringing this sample up, you should notice the log from the correlator
container telling you "Hello world!".

Sample Expected Output:
```

> application_1  | 2019-06-20 15:31:50.356 ##### [140502313332608] - Correlator, version 10.3.1.1.349560, running

Reviewer: ```suggestion
```

application_1  | 2019-06-20 15:31:50.357 INFO  [140501802022656] - Added monitor HelloWorld
application_1  | 2019-06-20 15:31:50.357 INFO  [140501802022656] - Injected MonitorScript from file /apama_work/Simple/HelloWorld.mon (227ec20012df1a8d7263f9ee56b78cb9), size 770 bytes, compile time 0.00 seconds
application_1  | 2019-06-20 15:31:51.405 INFO  [140501751678720] - HelloWorld [1] Hello world! The current epoch is 1561044711.354903
```

Troubleshooting Docker-Compose
=======

> You may have to remove docker-compose defaults located at  .docker/config.json to ensure the project builds in the default manner. 

Reviewer: ```suggestion
Please make sure that the contents of .docker/config.json are correct and pointing to the right docker host or you may have problems building. 
```
